test/test_runcircuit.jl

```diff
@@ -1,4 +1,4 @@
-using PastaQ
+xusing PastaQ
```

Comment:
Same issue.

Reply:
Arggg. I grepped for this now and removed them.